lib/src/package_config_impl.dart

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import "package_config.dart";
+export "package_config.dart";
```

Comment:
https://dart.dev/guides/language/effective-dart/style#do-specify-exports-in-a-separate-section-after-all-imports

Reply:
Done.